data/apis/amberdata/beacons/amberdata aave_btc.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Amberdata AAVE/BTC",
+  "description": "Amberdata - AAVE/BTC - Pair",
+  "beaconId": "0x0259c6fa3994f392260869cbef72b2c9df45b563b8e78e5c9851a8183b51adc0",
+  "airnodeAddress": "0xF0f875aa568521FBd5246fAE7260E04d9ED6511b",
+  "templateId": "0xfce8a7c37dd14e010cd5ef02d5b0a87deedba396fcb4b75deb9ab4aed34017b5",
+  "chains": {
+    "rsk": {
+      "active": true,
+      "sponsor": "0xAb770D6E94e26E9388db8d6D10a2458973a7Ff8f",
+      "topUpWallets": [{ "walletType": "API3", "address": "0x25B246C3bA7B7353e286859FaE8913600b96B719" }],
```

Comment:
@aquarat Is `0x25B246C3bA7B7353e286859FaE8913600b96B719` derived from the mnemonic of `0xAb770D6E94e26E9388db8d6D10a2458973a7Ff8f`? [This](https://github.com/api3dao/operations/blob/main/src/generate-topup-wallets.ts#L22) has you enter it so I couldn't tell

Reply:
Never mind, I read the Airseeker code.